www/search.py

```diff
@@ -165,21 +240,22 @@ def GetResult(request):
 #request={'_': '1630833475755', 'callback': 'processData', 'images': 'on', 'nosensitive': 'on', 'springname': '*'}
 
 #request = {"md5": "311d2bc8fd1bdb092b7d1f162da5fc44"}
-
 for k,v in cgi.parse().items():
 	if isinstance(v, list):
 		request[k] = v[0]
 	else:
 		request[k] = v
 
-result = GetResult(request)
-
-if "callback" in request:
-	# strip anything except a-Z0-9
-	print("Content-type: application/javascript\n")
-	cb = upqdb.escape(request["callback"], set("abcdefhijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"));
-	print( cb + "(" +  json.dumps(result) + ");")
+if "printMapKeywordList" in request:
```

Comment:
IMHO not needed, see above

Reply:
replied above. There's a need to get a fresh list of maps and keywords from the DB.

If there's a txt with the keywords, new maps may have been added since the txt was last updated.